app/retry.js

```diff
@@ -1,6 +1,6 @@
 const { totalRetries, retryInterval } = require('./config')
 
-async function retry (fn, retriesLeft = totalRetries, interval = retryInterval, exponential = true) {
+async function retry (fn, retriesLeft = totalRetries, interval = retryInterval, exponential = false) {
```

Comment:
I don't think I've seen an async function defined like this before

Reply:
Changed to be consistent.